medusa/indexers/indexer_base.py

```diff
@@ -243,7 +243,7 @@ def _save_images(self, sid, images):
                 if not len(images[image_type]):
                     continue
                 merged_image_list = [y[1] for y in [next(iteritems(v)) for _, v in iteritems(images[image_type])]]
-                highest_rated = sorted(merged_image_list, key=lambda k: k['rating'], reverse=True)[0]
+                highest_rated = sorted(merged_image_list, key=lambda k: float(k['rating']), reverse=True)[0]
```

Comment:
Yes, it will accept number or '1', '2.1', as it's cast to float. This code has been in here for last 3 weeks. Havent' seen any issues regarding that.